<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Dead</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Dead</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="dead_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>S-DEAD</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="sdead_combobox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="partition_dead_checkbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>PARTITION</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="partition_dead_combobox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Live</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>L-RED</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Live</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="lred_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="live_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="partition_live_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="partition_live_checkbox">
          <property name="text">
           <string>PARTITION</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Live-0.5</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QComboBox" name="live5_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Live_0.5</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Roof</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_9">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>LROOF</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="lroof_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>SNOW</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="snow_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>EV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ev_combobox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Acc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="acc">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>0.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.35</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="importance_factor">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>0.8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.4</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="export_to_etabs_button">
       <property name="toolTip">
        <string>Export Selected Load Combinations To ETABS Model</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/etabs.png</normaloff>:/civiltools/images/etabs.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="toolTip">
        <string>Export Selected Load Combinations To ETABS Model</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/cancel.svg</normaloff>:/civiltools/images/cancel.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="self_weight">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Self Weight</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="replace">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Cantilever Frames/Floors</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="etabs_cantilever">
          <property name="text">
           <string>ETABS</string>
          </property>
          <property name="icon">
           <iconset resource="../../civiltools.qrc">
            <normaloff>:/civiltools/images/etabs.png</normaloff>:/civiltools/images/etabs.png</iconset>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="freecad_cantilever">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>FreeCAD</string>
          </property>
          <property name="icon">
           <iconset resource="../../civiltools.qrc">
            <normaloff>:/civiltools/images/freecad.svg</normaloff>:/civiltools/images/freecad.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_to_etabs_button_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Export Selected Load Combinations To ETABS Model</string>
       </property>
       <property name="text">
        <string>Set Selections As cantilever</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/freecad.svg</normaloff>:/civiltools/images/freecad.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="help">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/help.png</normaloff>:/civiltools/images/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
